extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -299,56 +286,90 @@ export class SchemaCompareMainWindow {
 				'operationId': this.comparisonResult.operationId
 			}).send();
 
-		let data = this.getAllDifferences(this.comparisonResult.differences);
-
-		this.differencesTable.updateProperties({
-			data: data,
-			columns: [
-				{
-					value: loc.type,
-					cssClass: 'align-with-header',
-					width: 50
-				},
-				{
-					value: loc.sourceName,
-					cssClass: 'align-with-header',
-					width: 90
-				},
-				<azdata.CheckboxColumn>
-				{
-					value: loc.include,
-					cssClass: 'align-with-header',
-					width: 60,
-					type: azdata.ColumnType.checkBox,
-					action: azdata.ActionOnCellCheckboxCheck.customAction
-				},
-				{
-					value: loc.action,
-					cssClass: 'align-with-header',
-					width: 30
-				},
-				{
-					value: loc.targetName,
-					cssClass: 'align-with-header',
-					width: 150
-				}
-			],
-			CSSStyles: { 'left': '15px' },
-			width: '98%'
-		});
-
-		this.splitView.addItem(this.differencesTable);
-		this.splitView.addItem(this.diffEditor);
-		this.splitView.setLayout({
-			orientation: 'vertical',
-			splitViewHeight: 800
-		});
-
 		this.flexModel.removeItem(this.loader);
 		this.flexModel.removeItem(this.waitText);
 		this.resetButtons(ResetButtonState.afterCompareComplete);
 
 		if (this.comparisonResult.differences.length > 0) {
+			const data = this.getAllDifferences(this.comparisonResult.differences);
```

Comment:
Also made it so that we only create all this stuff if there are actual differences (since otherwise we don't show the table/editor anyways)

Reply:
makes sense. UI load time wise I am guessing this wont have any impact to show data after the wait loader is removed... right?

Reply:
It will likely be slightly slower due to having to recreate all the components. But that typically isn't the slowest part of the logic so shouldn't have much customer impact.